test/classes/ferguson/delete-free/owned-demo.chpl

```diff
@@ -56,28 +56,28 @@ proc examples() {
   var owned5 = new owned C(5)?;
   writeln("owned5 = ", owned5.borrow());
 
-  // The clear method can be used to delete the
+  // The release function can be used to delete the
   // managed object, if any.
-  owned5.clear(); // deletes C(5)
-  writeln("after owned5.clear()");
+  delete owned.release(owned5); // deletes C(5)
+  writeln("after delete owned.release(owned5)");
   writeln("owned5 = ", owned5.borrow());
 
-  // The retain method can be used to provide a new value for
+  // The adopt function can be used to provide a new value for
   // an owned to point to.
   // After this line:
   //  'owned1' will contain C(100); previous object C(4) will be destroyed
-  owned1.retain(new unmanaged C(100));
-  writeln("after owned1.retain C(100)");
+  owned1 = owned.adopt(new unmanaged C(100));
+  writeln("after owned1.adopt C(100)");
   writeln("owned1 = ", owned1.borrow());
 
-  // The release method can be used to extract the
+  // The release function can be used to extract the
```

Comment:
It's in a test's comment, so doesn't really matter, but `method` or `type method` seems more accurate than `function` to me here...?